.github/workflows/ca-update.yaml

```diff
@@ -0,0 +1,30 @@
+name: CA Update
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 2 * * 0"
+jobs:
+  Step-1:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10' 
+        
+      - name: Download and Parse
+        run: |
+          python3.10 -m pip install cryptography
+          wget https://curl.se/ca/cacert.pem
+          python3.10 Meta/ca-update.py
+          mv ca_certs.ini Base/etc/
+          rm -f cacert.pem
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: 'Update `certs.ini`'
```

Comment:
"Update `certs.ini`" would not be accepted as a Serenity commit message. Also will this only make a PR if something has changed?

Reply:
There are a few problems right now and it is therefore a Draft. But I wanted to have some discussion baseline for it.